vividus-plugin-csv/src/main/java/org/vividus/bdd/expression/GenerateFileExpressionProcessor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.expression;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.vividus.csv.CsvReader;
+import org.vividus.util.ResourceUtils;
+import org.vividus.util.parquet.ParquetUtils;
+
+public class GenerateFileExpressionProcessor implements IExpressionProcessor
+{
+    private static final Pattern GENERATE_PARQUET_PATTERN = Pattern
+            .compile("^generateParquetFromCsv\\(((.*\\/)*(.*)\\.csv), (.*)\\)$");
+
+    private static final int CSV_PATH_GROUP = 1;
+    private static final int CSV_NAME_GROUP = 3;
+    private static final int SCHEMA_PATH_GROUP = 4;
+
+    private final CsvReader csvReader = new CsvReader();
+
+    @Override
+    public Optional<String> execute(String expression)
+    {
+        Matcher generateParquetMatcher = GENERATE_PARQUET_PATTERN.matcher(expression);
+        if (generateParquetMatcher.find())
+        {
+            File temporaryFile = null;
+            String csvPath = generateParquetMatcher.group(CSV_PATH_GROUP);
+            String csvName = generateParquetMatcher.group(CSV_NAME_GROUP);
```

Comment:
I recommend to use `org.apache.commons.io.FilenameUtils.getBaseName(String)` to extract file name